src/proteomes/config/__tests__/__snapshots__/ProteomesColumnConfiguration.spec.tsx.snap

```diff
@@ -74,13 +74,205 @@ exports[`ProteomesColumnConfiguration component should render column "components
     <li>
       Chromosome 1
     </li>
+    <li>
+      Chromosome 1
+    </li>
+    <li>
+      Chromosome 1
+    </li>
+    <li>
+      Chromosome 1
+    </li>
+    <li>
+      Chromosome 1
+    </li>
+    <li>
+      Chromosome 1
+    </li>
+    <li>
+      Chromosome 2
+    </li>
+    <li>
+      Chromosome 2
+    </li>
+    <li>
+      Chromosome 2
+    </li>
+    <li>
+      Chromosome 3
+    </li>
+    <li>
+      Chromosome 3
+    </li>
+    <li>
+      Chromosome 4
+    </li>
+    <li>
+      Chromosome 4
+    </li>
+    <li>
+      Chromosome 5
+    </li>
+    <li>
+      Chromosome 5
+    </li>
+    <li>
+      Chromosome 6
+    </li>
+    <li>
+      Chromosome 7
+    </li>
+    <li>
+      Chromosome 8
+    </li>
+    <li>
+      Chromosome 9
+    </li>
+    <li>
+      Chromosome 9
+    </li>
+    <li>
+      Chromosome 9
+    </li>
+    <li>
+      Chromosome 9
+    </li>
+    <li>
+      Chromosome 9
+    </li>
+    <li>
+      Chromosome 10
+    </li>
+    <li>
+      Chromosome 10
+    </li>
+    <li>
+      Chromosome X
+    </li>
+    <li>
+      Chromosome 11
+    </li>
+    <li>
+      Chromosome 11
+    </li>
+    <li>
+      Chromosome 12
+    </li>
+    <li>
+      Chromosome 13
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 14
+    </li>
+    <li>
+      Chromosome 15
+    </li>
+    <li>
+      Chromosome 15
+    </li>
+    <li>
+      Chromosome 16
+    </li>
+    <li>
+      Chromosome 16
+    </li>
+    <li>
+      Chromosome 17
+    </li>
+    <li>
+      Chromosome 17
+    </li>
+    <li>
+      Chromosome 17
+    </li>
+    <li>
+      Chromosome 17
+    </li>
+    <li>
+      Chromosome 18
+    </li>
+    <li>
+      Chromosome 19
+    </li>
+    <li>
+      Chromosome 20
+    </li>
+    <li>
+      Chromosome 21
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome 22
+    </li>
+    <li>
+      Chromosome Y
+    </li>
+    <li>
+      Chromosome Y
+    </li>
+    <li>
+      Mitochondrion MT
+    </li>
+    <li>
+      Unplaced
+    </li>
     <li>
       <button
         class="button tertiary expandable-list__action"
         data-testid="expandable-message"
         type="button"
       >
-        64 more components
+        Fewer components
```

Comment:
This is because of the change in src/shared/__test-helpers__/testColumnConfiguration.tsx that is wrapping the click on the button with `act()`, otherwise the snapshot won't be reflecting the expanded state

Reply:
Handled in `main` through https://github.com/ebi-uniprot/uniprot-website/commit/3a74f6435b211135576d5d44e44ae03447f0d783 and the code will be reverted in https://www.ebi.ac.uk/panda/jira/browse/TRM-31656